Header header
float64[] finger_joint
float64[] direction
float64[] palm_normal
float64[] wrist_position
float64[] palm_position